graylog2-server/src/main/java/org/graylog2/rest/resources/search/AbsoluteSearchResource.java

```diff
@@ -90,6 +92,7 @@ public SearchResponse searchAbsolute(
             @ApiParam(name = "limit", value = "Maximum number of messages to return.") @QueryParam("limit") int limit,
             @ApiParam(name = "offset", value = "Offset") @QueryParam("offset") int offset,
             @ApiParam(name = "filter", value = "Filter") @QueryParam("filter") String filter,
+            @ApiParam(name = "streams", value = "Comma separated list of streams to search in") @QueryParam("streams")  String streams,
```

Comment:
I may be a little bit to picky, but I would replace `Comma separated list of streams to search in` with `Comma separated list of stream IDs to search in`, to explicitly mention we are talking about stream IDs here...